roxie/roxiemem/roxiemem.cpp

```diff
@@ -8425,8 +8425,6 @@ class RoxieMemTests : public CppUnit::TestFixture
 
         for (unsigned pass=0; pass < 8; pass++)
         {
-            //numPagesAfterCleanup() also has the side-effect of freeing up any empty pages.
```

Comment:
This looks like it should be retained.  It may have a side-effect used in the test.